datashare-app/src/main/java/org/icij/datashare/tasks/TaskManagerRedis.java

```diff
@@ -130,6 +132,9 @@ public void error(String taskId, Throwable reason) {
         tasks.put(taskId, taskView);
     }
 
+    @Override
+    public void close() throws IOException {}
```

Comment:
Nothing to do with Redis to close ongoing tasks?

Reply:
see https://github.com/ICIJ/datashare/pull/1284/commits/ffaafe117be48a2e525ff8a9123cdf144f2afeb0